README.rst

```diff
@@ -145,9 +144,11 @@ as administrator.
 Run tests
 =========
 
-If you have followed the steps above to set up your environment for testing, running the test suite should be as easy as running nosetests(1) with the following options::
+If you have followed the steps above to set up your environment for testing,
+running the test suite should be as easy as running
+`pytest <http://pytest.org/>`_ with the following options::
```

Comment:
I think in the environment we would also have to install pytest. Run 

    pip install pytest

(I think it was not installed before)

Reply:
actually it is part of `tests/requirements.txt`. I added a link to the installation of the testing environment in 00d3330